scripts/exregional_run_analysis.sh

```diff
@@ -157,6 +157,17 @@ YYYYMMDD=${YYYYMMDDHH:0:8}
 #
 #-----------------------------------------------------------------------
 #
+# For running observers,  extract the valid time of the restart files
+#
+#-----------------------------------------------------------------------
+#
+if [ ${DO_GSIOBSERVER} == "TRUE" ]; then
+  HH=${ANAL_DATE:8:2}
+  YYYYMMDD=${ANAL_DATE:0:8}
+fi
```

Comment:
if we only run one time level, then we may not need this part. CDATE is already available in the script.